apps/masterbots.ai/components/routes/thread/user-thread-panel.tsx

```
@@ -167,37 +173,37 @@ export default function UserThreadPanel({
-      handleThreadsChange();
+      handleThreadsChange()
     }
-  }, [activeCategory, activeChatbot, isOpenPopup, pathname]);
+  }, [activeCategory, activeChatbot, isOpenPopup, pathname])
 
   useEffect(() => {
     if (
       threads &&
       threads.filter(t => t.threadId === activeThread?.threadId).length
-    ) return
+    )
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)